### ISSUE TITLE
Belt not deconstructing nor constructing

### ISSUE THREAD

**Issue by knmh84** [MaFi-Games/Captain-of-Industry-issues#169]
the belts from iron and concrete are stuck they cannot be constructed nor deconstructed
[22-06-01_04-16-40_7254-autosave.zip](https://github.com/MaFi-Games/Captain-of-Industry-issues/files/8809996/22-06-01_04-16-40_7254-autosave.zip)

**Comment by lowstrife** [comment]
Ran into the same issue. Was modifying existing belts as I realized Potatoes need U-shape conveyors to transport them. 

I am not 100% sure, but I believe the bug started when I used SHIFT+click to delete an entire conveyor belt. 

My save game currently can't construct any belts, pipes or buildings. Construction (or deconstruction) will not progress past 0%, even when using unity to complete it instantly. 

[Belt Issue.zip](https://github.com/MaFi-Games/Captain-of-Industry-issues/files/8810172/Belt.Issue.zip)
